tests/cloud_functions/token_refresh/conftest.py

```
@@ -0,0 +1,27 @@
+"""Conftest for token refresh tests."""
+import pytest
+
+
+class MockRequest:
+    """Mock request for testing."""
+
+    method = "POST"
+    headers = {"Authorization": "Bearer test_token"}
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)